exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ParameterizationTests.kt

```diff
@@ -18,7 +18,7 @@ class ParameterizationTests : DatabaseTestsBase() {
     }
 
     private val supportMultipleStatements by lazy {
-        setOf(TestDB.MYSQL, TestDB.MARIADB, TestDB.POSTGRESQL, TestDB.SQLSERVER)
+        setOf(TestDB.MYSQL_V5, TestDB.MARIADB, TestDB.POSTGRESQL, TestDB.SQLSERVER)
```

Comment:
`allowMultiQueries` should definitely be supported by MySQL8. If the test is failing, it needs to be checked not excluded.